src_test/ppx_deriving_sexp/dune

```diff
@@ -1,6 +1,13 @@
-(test
+(executable
```

Comment:
For whatever reason, it turns out `enabled_if` doesn't work with `test` in dune 3

Reply:
Or a least it still tries to build which leads to an [error](https://github.com/ocaml-ppx/ppx_import/actions/runs/3543658771/jobs/5950299182) because `ppx_sexp_conv` is not installed.